bin/verify-exercises

```diff
@@ -1,40 +1,86 @@
 #!/usr/bin/env bash
 
 # Synopsis:
-# Test the track's exercises.
-# 
-# At a minimum, this file must check if the example/exemplar solution of each 
-# Practice/Concept Exercise passes the exercise's tests.
-#
-# To check this, you usually have to (temporarily) replace the exercise's solution files
-# with its exemplar/example files.
-#
-# If your track uses skipped tests, make sure to (temporarily) enable these tests
-# before running the tests.
-#
-# The path to the solution/example/exemplar files can be found in the exercise's
-# .meta/config.json file, or possibly inferred from the exercise's directory name.
+# Verify that each exercise's example/exemplar solution passes the tests.
+# You can either verify all exercises or a single exercise.
 
 # Example: verify all exercises
-# ./bin/verify-exercises
+# bin/verify-exercises
```

Comment:
Doesn't this assume `.` is in `PATH`?

Reply:
No. It's a relative path, no PATH lookup.